pkg/plugins/pfs/decl.go

```diff
@@ -15,6 +15,7 @@ type PluginDecl struct {
 	PluginPath      string
 	PluginMeta      plugindef.PluginDef
 	KindDecl        kindsys.Def[kindsys.ComposableProperties]
+	GrafanaVersion  string
```

Comment:
Could we put this...anywhere else? Like directly on a jenny, maybe? It's OK if not, this type is going to need to change anyway. But it mixes core-specific logic here, which is an incremental step away from the goal of avoiding assuming that core exists in pfs code.

If it's too hard to find another spot, let's at least add a `TODO specific to core, try to remove when refactoring` comment here and on other lines where we're using it

Reply:
I'm going to move to the jenny directly. It was here because at the beginning I set the version in core plugins too and it isn't necessary.

Reply:
Done :)